StereoKitC/asset_types/mesh.cpp

```diff
@@ -437,6 +437,18 @@ bool32_t mesh_ray_intersect(mesh_t mesh, ray_t model_space_ray, ray_t *out_pt, i
 
 	return nearest_dist != FLT_MAX;
 }
+///////////////////////////////////////////
+
+bool32_t mesh_get_triangle(mesh_t mesh, uint32_t triangle_index, vert_t* a, vert_t* b, vert_t* c) {
+	if (mesh->ind_count > triangle_index) {
```

Comment:
This should also validate that the `discard_data` flag is not set first. Meshes can be told to delete CPU side data for performance reasons, at the cost of not being able to do stuff like this. See `_mesh_set_skin` for a sample of this.

Reply:
It logs and returns false if the mesh is discard